src/adhocracy_core/adhocracy_core/resources/subscriber.py

```diff
@@ -244,21 +245,35 @@ def send_password_reset_mail(event):
     event.registry.messenger.send_password_reset_mail(user, password_reset)
 
 
-def send_activation_mail_or_activate_user(event):
-    """Send mail with activation link if a user is created.
+def apply_user_activation_configuration(event):
+    """Activate user or send activation or invite email.
 
     If the setting "adhocracy.skip_registration_mail" is true, no mail is send
     but the user is activated directly.
     """
-    settings = event.registry.settings
     user = event.object
-    skip_mail = asbool(settings.get('adhocracy.skip_registration_mail', False))
-    if skip_mail:
+    registry = event.registry
+    sheet = registry.content.get_sheet(user, IActivationConfiguration)
+    activation_config = sheet.get()['activation']
+    if _is_activation_disabled(registry):
         user.activate()
```

Comment:
skip_registration_mail was only a workaround for thentos. It makes more sense to add this option to the activationConfiguration sheet, then we can also add users with the sdi without activation mail.